build/generate-manifests.sh

```diff
@@ -40,25 +40,7 @@ fi
 mkdir -p "$BUNDLE_DIR"
 
 function checkDependencies() {
-  if ! [ -x "$(command -v jq)" ]; then
-    echo "Please install jq package.'"
-    exit 1
-  fi
-
-  # Looks for yq-python (https://github.com/kislyuk/yq) first, as installed in CI image, fallbacks to yq.
-  # Note there's another yq package, written in golang, which is not the one we want: https://github.com/mikefarah/yq
-  YQ=$(which yq-python 2>/dev/null || which yq 2>/dev/null || echo "")
-
-  if ! [ -x "$(command -v ${YQ})" ]; then
-    echo "Please install the python yq package, e.g. 'pip install --user yq'"
-    exit 1
-  else
-    s="yq 2.*"
-    if ! [[ $(${YQ} --version) =~ $s ]]; then
-      echo "Install the correct (python) yq package, e.g. 'pip install --user yq'"
-      exit 1
-    fi
-  fi
+  :
```

Comment:
You can remove this function

Reply:
I left it in in case we need to add dependencies later. But I'll remove it.